tests/sanity/tests/tracker/issues.spec.ts

```diff
@@ -26,7 +26,7 @@ test.describe('Tracker issue tests', () => {
       createLabel: true,
       labels: `CREATE-ISSUE-${generateId()}`,
       component: 'No component',
-      estimation: '2',
+      estimation: '2.5',
```

Comment:
Not sure, if it is a good change. It kinda does test new minutes presentation, but ignores a 0 minutes case. Should I rewrite tests to create an issue with `2h`, then change it to `2h 30m`, and then change it to `8h`?

Reply:
"Should I rewrite tests to create an issue with 2h, then change it to 2h 30m, and then change it to 8h?" - Yes, it would be fine